src/mapgen.cpp

```diff
@@ -3279,26 +3278,32 @@ C..C..C...|hhh|#########\n\
                 rotate(3);
             }
         } else if (tw != 0 || rw != 0 || lw != 0 || bw != 0) { // Sewers!
+
+            const auto is_lab = []( const oter_id &id ) {
```

Comment:
No, not really. I was getting rid of unreliable operators (see eb653822d4264fc35809f3a00fc46212082c964f). 
#14136 is a separate issue, I didn't mean mending it here (to stick with the pure refactoring).